### PR TITLE
LPS-122740 SF

### PR DESCRIPTION
**Pull request by CarlosBrichete**

### CODE REVIEW

**Comment by CarlosBrichete** [comment]
ci.test:sf

**Comment by CarlosBrichete** [comment]
ci:test:relevant

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
@manoelcyreno

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+			vocabularyName = "Audience");
+
+		var categoryId2 = JSONCategory.addCategory(
+			groupName = "Global",
+			title = "Stage 1",
+			vocabularyName = "Stage");
+	}
+
+	tearDown {
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboardAuditGraph.testcase:29]: Line 29 until 37.
Take a look in other poshi tests, if it is inside of the else or not, please.

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
ci:test:relevant

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**